jmist-core/src/main/java/ca/eandb/jmist/math/Spheroid.java

```diff
@@ -516,4 +516,13 @@ public void writeObjToStream(int stacks, int slices, PrintStream out) {
     }
   }
 
+  public Box3 boundingBox(){
+    double rx = Math.sqrt(Math.pow(a * basis.u().x(), 2.0) + Math.pow(a * basis.v().x(), 2.0) + Math.pow(c * basis.w().x(), 2.0));
```

Comment:
Maybe throw in a `private static double sqr(double x)` method that returns `x * x`. That way we just use multiplication instead of the more general (and more expensive) `pow()`.